### DISCUSSION TITLE
Remove obscene words from your dictionary

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1579] [Q&A]
Hello!
I had found the following obscene words in your [dictionary](https://github.com/simple-login/app/blob/master/local_data/words.txt):

```
yatsuka
isuka
tsukada
suka
otsuka
idiot
pizdoblyadst
```

They are insults in some languages. Really strong insults. Please remove them and check your dictionary with this list of obscene insults:
[OBSCENE WORDS](https://pastebin.com/f6gTnMYC)

**Comment by nguyenkims** [comment]
Thanks for reporting. We'll change the list of words we use for alias generation soon, probably based on what's suggested here https://github.com/simple-login/app/discussions/1578 which will avoid the issue of bad words.
